B-1B-set.xml

```diff
@@ -67,7 +67,7 @@ B-1B set file.
             </maintainer>
         </maintainers>
         <aero>B-1B-yasim</aero>
-        <fuel-fraction>0.65</fuel-fraction>
+        <fuel-fraction>0.00</fuel-fraction>
```

Comment:
Not sure about this. I like the idea of having some realistic complexity to the aircraft, but it's very common across aircraft in flightgear to start with fuel loaded. Open to discussion.